features/paint-order.yml

```
@@ -0,0 +1,7 @@
+name: paint-order
+description: "The `paint-order` CSS property sets whether the `stroke` is above or below the `fill`. In SVG content, `markers` can also be ordered. For example, `paint-order: markers stroke fill;` will draw the markers, then stroke, then fill."
```

Edits here with these motivations:

- I read "above or below" as meaning the vertical axis and I've made an attempt to clear this up. I don't know if I succeeded.
- Using values as both literals and words strikes me as a bold stylistic choice. Since we don't really know where these things will end up, I'm taking a more conservative approach.
- I should probably write it up in the guidelines, but we ought to prefer the present tense whenever it makes sense.


```suggestion
description: "The `paint-order` CSS property sets the z-order of strokes, fills, and (in SVG content) markers. For example, `paint-order: markers stroke fill;` draws the markers, then stroke, then fill."
```

---

I think this is an improvement. I am on the fence on whether it is clear that the later layers might cover up earlier layers, as it is reliant on the "paint" metaphor, and not explicitly stated.  Thoughts?